### ISSUE TITLE
Power apps (V2) label is not associated with the list items

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#4391]
### Describe the issue 

Token picker section header component is not associated with section list items

1. Open designer standalone (nx serve). 
2. Create a flow with more than one token in token picker. 
3. Use screen reader of choice to tab over the list items. 
4. Observe that list items are not logically correlated with their labels. 

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

![image](<https://github.com/Azure/LogicAppsUX/assets/10837129/b788860e-1335-4eae-9287-259d209f0515>)

### Browser 

Edge

### Additional context 

ADO link: [https://msazure.visualstudio.com/One/\_workitems/edit/26878925](<https://msazure.visualstudio.com/One/_workitems/edit/26878925>)
